service/r2/store/mock.go

```diff
@@ -0,0 +1,212 @@
+// Automatically generated by MockGen. DO NOT EDIT!
```

Comment:
Hm strange that there is no license header here. How did you generate this? Did you use `make mock-gen`?

Reply:
Looks like my local version of node is from 2014 or something. That seemed to cause issues, Ill make sure that is working.